### PR TITLE
Improved docs for `fetchModel()`

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: en/controllers.rst [hunk 1]
```diff
@@ -463,6 +449,25 @@ the controller's default one::
         )
         ->all();
 
+.. php:method:: fetchModel(string $alias, array $config = [])
```
Comment (othercorey) [inline en/controllers.rst]: This doesn't seem to be the signature used in the examples.

File: en/controllers.rst [hunk 2]
```diff
@@ -463,6 +449,25 @@ the controller's default one::
...
+
+The ``fetchModel()`` method is useful to load non ORM models or ORM tables that
+are not the controller's default::
```
Comment (othercorey) [inline en/controllers.rst:455]: > or ORM tables that are not the controller's default

This is the same definition as fetchTable() which is confusing. We should describe fetchModel() for just one use case.

File: en/controllers.rst (hunk 1 above)
Comment (ADmad) [inline en/controllers.rst]:
```suggestion
.. php:method:: fetchModel(string|null $modelClass = null, string|null $modelType = null)
```

File: en/controllers.rst (hunk 2 above)
Comment (ADmad) [reply en/controllers.rst:455]: Well "non ORM models" is the different bit here. This method can fetch any type of model including tables, which is what you get if you don't specify the 2nd argument.

**Pull request merged**